skbio/sequence/tests/test_sequence.py

```diff
@@ -82,8 +88,15 @@ def test_init_empty_sequence(self):
             self.assertEqual(seq.values.dtype, '|S1')
             self.assertEqual(seq.values.shape, (0, ))
             npt.assert_equal(seq.values, np.array('', dtype='c'))
-            self.assertEqual('', str(seq))
-            self.assertEqual(0, len(seq))
+            self.assertEqual(str(seq), '')
```

Comment:
Is this order (observed, expected) our convention in general or did you just change it to be consistent with the neighboring code?